docs/AssetsApi.md

```diff
@@ -37,7 +37,7 @@ MuxRuby.configure do |config|
 end
 
 api_instance = MuxRuby::AssetsApi.new
-create_asset_request = MuxRuby::CreateAssetRequest.new # CreateAssetRequest | 
+create_asset_request = {"input":"https://muxed.s3.amazonaws.com/leds.mp4","playback_policy":["public"]} # CreateAssetRequest | 
```

Comment:
I'm not against using an actual video here, but we might want to not use an MP4 URL directly, right? Should we put this behind cloudfront just to be safe in case someone decides to share this link and we get S3 pricing?

Reply:
Or, better yet, shouldn't we use a non-existent URL, like we do for tracks below?